.github/workflows/ci_macos.yml

```diff
@@ -2,14 +2,15 @@ name: Apple macOS
 
 on:
   push:
-    branches: [ "master" ]
+    branches: [ "*" ]
```

Comment:
The GitHub actions take up a good amount of time and I think failed jobs will always show up in the badge on the front  page, which is not really necessary for experimental/side branches. So I'd like to keep it to the `master` branch for now. If we open this up to other branches, I would rather do it in one commit for all CI jobs.

Reply:
Ok, it was just to experiment on my fork, I will revert it